lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -402,23 +408,20 @@
             text-overflow: ellipsis;
             white-space: nowrap;
 
-            .adf-datatable-cell, .adf-datatable-cell-header {
+            &.adf-datatable-cell-header,
```

Comment:
Can we double check that these changes keep the usability and the accessibility for the cells.
In this PR the cell is not clickable on its entire area, only the texts. The borders need to be applied to the whole area of the cell and not only the text.

Reply:
I've noticed that behavior too, but on development, that is why I thought that was expected...